vechical_tracker.py

```diff
@@ -0,0 +1,114 @@
+import cv2
+import fire
+import imutils
+from tracking.mathhelper import get_bb_from_centroid
+from tracking.v2.optical_flow_tracker import OpticalFlowTracker
+from tracking.v2.iou_tracker import IOUTracker
+from tracking.v2.py.trackableobject import TrackableObject
+from tracking.v2.retina_net_camera import RetinaNetCamera
+import numpy as np
+
+trackable_objects = {}
+
+labels_to_names = {0: 'Biker',
+                   1: 'Car',
+                   2: 'Bus',
+                   3: 'Cart',
+                   4: 'Skater',
+                   5: 'Pedestrian'}
+
+
+def create_new_trackers(preds, frame, frame_number, trackers):
+    boxes, scores, labels, = preds
+
+    for i, (box, score, label) in enumerate(zip(boxes[0], scores[0], labels[0])):
+        if(score < 0.50):
+            continue
+        if label == 5:
+            continue
+    
+
+        tracker = OpticalFlowTracker()
+        # label = "{}_{}_{}".format(labels_to_names[label], frame_number, i)
+        label = "{}".format(labels_to_names[label])
+        color = get_random_color()
+
+        tracker.start_track(frame, box, label)
+        tracker.assign_color(color)
+        trackers.append(tracker)
```

Comment:
@spaudel do you think using one [optical flow](https://github.com/nishontan/vechical-detect-and-track/blob/4-refactor-optical-flow-tracker/tracking/v2/optical_flow_tracker.py) instance per object is a good idea?